### PR TITLE
Modfqmmgr 237

### PR DESCRIPTION
**Pull request by kjain110**
## Purpose
_Describe the purpose of the pull request. Include background information if necessary._

## Approach
_How does this change fulfill the purpose?_

#### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

## Pre-Merge Checklist

If you are adding entity type(s), have you:
- [ ] Added the JSON5 definition to the `src/main/resources/entity-types` directory?
- [ ] Ensured that GETing the entity type at `/entity-types/{id}` works as expected?
- [ ] Added translations for all fields, per the [translation guidelines](/translations/README.md)? (Check this by ensuring `GET /entity-types/{id}` does not have `mod-fqm-manager.entityType.` in the response)
- [ ] Added views to liquibase, as applicable?
- [ ] Added required interfaces to the module descriptor?
- [ ] Checked that querying fields works correctly and all SQL is valid?

### CODE REVIEW

File: translations/mod-fqm-manager/en.json [hunk 1]
```diff
@@ -646,5 +646,18 @@
...
-  "entityType.drv_location_details.primary_service_point": "Primary service point"
+  "entityType.drv_location_details.primary_service_point": "Primary service point",
+  "entityType.drv_config_data_details": "Config data",
+  "entityType.drv_config_data_details.id": "ID",
+  "entityType.drv_config_data_details.code": "Code",
+  "entityType.drv_config_data_details.config_name": "Config name",
+  "entityType.drv_config_data_details.description": "Description",
+  "entityType.drv_config_data_details.value": "Value",
+  "entityType.drv_config_data_details.user_id": "User ID",
+  "entityType.drv_config_data_details.Created_date": "Created date",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:657]: Should this be capital?

Comment (kjain110) [reply translations/mod-fqm-manager/en.json:657]: Where @ncovercash

File: src/main/resources/entity-types/configuration/drv_config_data_details.json5 [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "config.jsonb ->> 'value'"
+    },
+    {
+      name: 'user_id',
+      dataType: {
+        dataType: 'stringType',
```
Comment (bvsharp) [inline src/main/resources/entity-types/configuration/drv_config_data_details.json5:103]: this should probably be rangedUUIDType (really it should be openUUIDType but we're not using that atm)

**Review by ncovercash — APPROVED** [review]

File: src/main/resources/entity-types/configuration/drv_config_data_details.json5 (hunk 2 above)
Comment (kjain110) [reply src/main/resources/entity-types/configuration/drv_config_data_details.json5]: @bvsharp  done

**Pull request merged**